public/js/models/projectsModel.js

```diff
@@ -5,19 +5,34 @@ var app = app || {};
 (function (module) {
   function Project (projectObj) {
     this.title = projectObj.title;
-    this.githubUrl = projectObj.githubUrl;
-    this.deploymentUrl = projectObj.deploymentUrl;
+    this.githubName = projectObj.githubName;
     this.imageUrl = projectObj.imageUrl;
     this.synopsis = projectObj.synopsis;
   }
 
   Project.projects = [];
 
   Project.loadProjects = function (callback) {
-    $.getJSON('/data/projects.json').then(function(data) {
+    $.getJSON('/data/projects.json').then(function (data) {
       data.map((project) => Project.projects.push(new Project(project)));
+      loadProjectsFromGithub(callback);
+    });
+  };
+
+  function loadProjectsFromGithub (callback) {
+    $.ajax({
+      url: '/github/user/repos',
+      method: 'GET'
+    })
+    .then(data => {
+      Project.projects.map(proj => {
+        let currentProject = data.filter(p => p.name === proj.githubName)[0];
```

Comment:
Your code is breaking here. If I console.log currentProject on the next line, I get undefined.